openedx/frontend-app-authoring
# feat: delete collection [FC-0062]

## Description

Adds delete option to collection card menu. It displays a confirmation box before deleting and a toast with button to undo delete action for 10 seconds.

## Supporting information

* Addresses: https://github.com/openedx/frontend-app-authoring/issues/1295
* `Private-ref`: [FAL-3851](https://tasks.opencraft.com/browse/FAL-3851)

## Testing instructions

* Setup tutor with this PR and https://github.com/open-craft/tutor-contrib-meilisearch plugin
* Create library and some collections in the library.
* Use delete option in collection card menu and verify its functionality.
* Use the undo button in the toast to restore collection.

## Concerns

Adding the undo button in a toast doesn't seem the best option in terms of accessibility.
